Makefile.include

```diff
@@ -223,9 +226,6 @@ ifneq ($(GNRC_NETIF_NUMOF),1)
   CFLAGS += -DGNRC_NETIF_NUMOF=$(GNRC_NETIF_NUMOF)
 endif
 
-# process provided features
-include $(RIOTMAKE)/features.inc.mk
```

Comment:
Do I understand correctly, that the `features.inc.mk` is not used anymore? In that case: shouldn't this PR also remove that file?

Reply:
yep, fixed